docs/docs/features/subscriptions.md

```diff
@@ -5,46 +5,55 @@ parent: Features
 nav_order: 3
 ---
 
-# Subscriptions (aka Topic-based Routing)
+# Subscriptions
 {: .no_toc }
 
 * toc
 {:toc}
 
 ## Introduction
 
-Subscriptions provide consumer applications a powerful mechanism to express
-interest in receiving only those messages which satisfy criteria specified by
-them.  In the absence of subscriptions, a consumer attached to a queue can
-receive any and all messages posted on the queue, and should be in a position
-to process all of them.  In other words, the queue is viewed as a logical
-stream of homogeneous data.  While this may work in some or most cases, there
-are scenarios where this restriction prevents a more flexible or natural
-arrangement of consumer applications.  For example, some users may prefer one
-set of consumers to handle messages of a certain type, and another set of
-consumers to handle messages of a certain other type.  This is where
-subscriptions come in -- they enable consumer applications to "subscribe" to
-messages of a certain type, thereby *logically* converting a queue into a
-stream of heterogeneous data.
+Subscriptions provide consumer applications a powerful mechanism to only
+receive the messages from a queue that match a specified expression.
+In essence, subscriptions allow the user to achieve topic-based message
+filtering and/or message routing.
+
+In the absence of subscriptions, a consumer attached to a queue can receive
+and should be able to process any and all messages posted on the queue. In
+other words, the queue is viewed as a logical stream of homogeneous data.
+While this may work in some or most cases, there are scenarios where this
+is limiting.
+
+For example, a user may prefer one set of consumers to handle messages of a
+certain type, and another set of consumers to handle messages of a certain
+other type. Or, a user may have a queue of messages that should all be
+processed by some consumer applications, but certain applications may only be
+interested in a certain subset of messages and want to ignore messages of a
+certain type. This is where subscriptions come in -- they enable consumer
+applications to "subscribe" to messages of a certain type and enable users
+to filter out messages for certain applications but not others, thereby
+*logically* converting a queue into a stream of heterogeneous data.
 
 Concretely speaking, producer applications can put any interesting message
 attributes in the *message properties* section of the message (*message
 properties* are a list of key/value pairs that a producer can associate with a
-message), and consumers can specify filters using one or more *message
-properties*.  For example, if a message contains these three properties:
+message), and consumers can filter messages using one or more of those *message
```

Comment:
"...and consumers can request BlazingMQ to filter..."

Reply:
Good spotting, will fix